src/test/sql/sql_translator_test.cpp

```diff
@@ -752,8 +752,8 @@ TEST_F(SQLTranslatorTest, WhereExists) {
 }
 
 TEST_F(SQLTranslatorTest, WhereNotExists) {
-  const auto actual_lqp =
-      compile_query("SELECT * FROM int_float WHERE EXISTS(SELECT * FROM int_float2 WHERE int_float.a = int_float2.a);");
+  const auto actual_lqp = compile_query(
+      "SELECT * FROM int_float WHERE NOT EXISTS(SELECT * FROM int_float2 WHERE int_float.a = int_float2.a);");
```

Comment:
how come you could add the NOT without altering the `_exists()` below?

Reply:
The question should be "how come it did not fail before?" :D
https://github.com/hyrise/hyrise/pull/1781/files#diff-fba0ebca5459aafe9c003ae4c3640a4dL36